src/zone.js

```diff
@@ -429,6 +430,102 @@ Zone.prototype.createDisk = function(name, config, callback) {
   );
 };
 
+/**
+ * Create an instance group manager in this zone.
+ *
+ * @see [InstanceGroupManager Resource]{@link https://cloud.google.com/compute/docs/reference/v1/instanceGroupManagers#resource}
+ * @see [InstanceGroupManagers: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/instanceGroupManagers/insert}
+ *
+ * @param {string} name - Name of the instance group manger.
+ * @param {config=} config - Configuration object.
+ * @param {instanceTemplate} config.instanceTemplate - Instance template to use for this instance group manager.
+ * @param {string} config.instanceTemplate - URL to an instance template to use for this instance group manager.
```

Comment:
I think we have to combine these params into one (unless you know otherwise? I'm not a JSDocs expert!):

```
@param {InstanceTemplate|string} config.instanceTemplate - Instance template to use for this instance group manager. If a string, it is treated as a complete URL to an instance template.
```